### PR TITLE
TSK-858: Send picture without text as comment for issues

### PR DESCRIPTION
**Pull request by ThetaDR**
# Contribution checklist

## Brief description
https://front.hc.engineering/workbench/platform/tracker/TSK-858

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: packages/text-editor/src/components/TextEditor.svelte [hunk 1]
```diff
@@ -50,7 +51,7 @@
     if (editor) editor.setEditable(editable)
   }
   export function submit (): void {
-    if (!editor.isEmpty) {
+    if (!editor.isEmpty || haveAttachment) {
       content = editor.getHTML()
       dispatch('content', content)
```
Comment (haiodo) [inline packages/text-editor/src/components/TextEditor.svelte]: Let's better send submit event with {isEmpty: boolean, content: content} and handle it properly in all use places.

Comment (haiodo) [reply packages/text-editor/src/components/TextEditor.svelte]: We already have isEmpty, so just handle empty submit inplace.

File: packages/text-editor/src/components/TextEditor.svelte [hunk 2]
```diff
@@ -31,6 +31,7 @@
   export let extensions: AnyExtension[] = []
   export let supportSubmit = true
   export let isEmpty = true
+  export let haveAttachment = false
```
Comment (haiodo) [inline packages/text-editor/src/components/TextEditor.svelte]: Why we need it?

Comment (ThetaDR) [reply packages/text-editor/src/components/TextEditor.svelte]: We don't. Forgot to remove this after reverting changes from first commit. Updated it

**Review by haiodo — APPROVED** [review]

**Pull request merged**